api/src/tests/test_context.rs

```diff
@@ -57,10 +57,13 @@ pub fn new_test_context(test_name: &'static str) -> TestContext {
         cached_framework_packages::module_blobs().to_vec(),
     )
     .unwrap()
-    .with_min_price_per_gas_unit(0)
-    .with_min_lockup_duration_secs(0)
-    .with_max_lockup_duration_secs(86400)
-    .with_initial_lockup_timestamp(0)
+    // not sure if this is intentional or needed?
+    .with_init_genesis_config(Some(Arc::new(|genesis_config| {
+        genesis_config.min_price_per_gas_unit = 0;
+        genesis_config.min_lockup_duration_secs = 0;
+        genesis_config.max_lockup_duration_secs = 86400;
+        // genesis_config.initial_lockup_duration_secs;
```

Comment:
what's this comment for?

Reply:
this is the only place that config logically changed. previously it used initial_lockup_timestamp = 0, which cannot be specified any more (as we use duration after current timestamp - so initial_lockup_timestamp cannot be before NOW).
If all tests pass, I'll make a comment that that was changed, otherwise I'll need to see to work around it.